index.js

```diff
@@ -71,7 +71,7 @@ const nightScoutHttpHeaders = {
     "Content-Type": "application/json",
 }
 
-const schedule = "*/5 * * * *";
+const schedule = "*/" + process.env.LINK_UP_TIME_INTERVAL + " * * * *";
```

Comment:
maybe add a `|| 5` for backwards compatibility reasons in case someone does not set the ENV variable.
the default value in the app.json works for Heroku but not for docker-compose or standalone deployments. :)

Reply:
done 👍🏻

Reply:
I've also added a default value (info) for LOG_LEVEL